text/XXX-A-Compilation-of-Short-Interactive-Tutorials-and-How-To-Guides-for-Coq.md

```diff
@@ -0,0 +1,257 @@
+The reference manual is by design not learning-oriented and not action-oriented, and it would be a mistake to try to bend it that way: it is not its purpose. It also provides relatively few examples (although many new examples have been added in the past few years and more would be welcome) and it is focused on Coq core features.
+Books like Coq'Art or Software Foundations provide nice pedagogical explanations but target specific audiences, and are often meant to be read from cover to cover. Moreover, they are not well suited to learn about specific features, to discover horizontally, and are not easy to keep updated.
+Other great resources exist like [Coq Tricks](https://github.com/tchajed/coq-tricks) or package-specific tutorials, but they are limited and are not necessarily easy to access for newcomers.
+
+
+### 2.2 Why it is important
+Not having a central collection of short and action-oriented documentation pieces is a real obstacle to getting more users outside of expert academic fields, and for Coq's popularity.
+
+#### Accessibility
+Having an easy to access documentation is of utmost importance to engage new users, and keep current users. Although it does work for some experts, we cannot expect from all potential new users to have to dig through a (possibly outdated) 600 pages book, or the reference manual to learn how to use a particular feature that appeals to them. Most particularly as these sources may not contain the basic answers they are looking for due to their nature. Similarly, having to say "try to look in the GitHub repository of the package, there might be some documentation there" is not possible if we wish for our software to be usable by all. Practical and pedagogical information must be easily accessible through a **nice** centralized online interface. This is especially true, at a time when new proof assistants emerge and attract new users outside computer science research.
+
+#### A documentation focused on practical usages
+Furthermore, not having such a documentation prevents people from actually learning by themselves, and discovering new amazing features of Coq that rival proof assistants may not have, as well as the richness of our ecosystem. While a lot of progress has been made on the ecosystem side with pages like [Awesome Coq](https://github.com/coq-community/awesome-coq) and the Coq Platform, there is still work to be done for the documentation that is hard to access. Many amazing features are still currently under-documented, and the existing documentation is often scattered out in the reference manual, books, or individual GitHub repositories, making it difficult to access. A symptom of that is the trouble that students are currently facing to discover new functionalities by themselves, compared to other software. In particular, in our experience, it is quite common that users (have to) rely on expert users they know to learn Coq.
```

Comment:
They already are some resources like [Coq Tricks](https://github.com/tchajed/coq-tricks) / [Awsome Coq](https://github.com/coq-community/awesome-coq) but it is by nature incomplete and the content is always what you want.

Moreover, it is a bit of a weak implementation because: 
- it is scattered across many websites, which are more or less stable (there is important turn over)
- more or less updated
- it creates duplication of effort as people can't check it all out

There is a reason we have a refman. Here it is the same but just for a different kind of documentation.  Information should be centralized, updated, easy to access and people can easily contribute if stuffs are missing.